components/sankeydiagram/customization/labels.md

```diff
@@ -0,0 +1,144 @@
+| `Position` | `string` <br/> (`inside`) | The position of the labels. The supported values are: <ul><li>`inside` - the label is positioned after the node, except for the nodes at the end of the Sankey, that are placed before the node;</li><li>`before` - the label is positioned before the node; </li><li>`after` - the label is positioned after the node; </li></ul> |
+| `Visible` | `bool` <br/> (`true`) | Whether the labels are visible. |
+
+
+## Nested Customization Tags
+
+The `<SankeyLabels>` tag exposes child tags for customization of the labels' border, margin, offset, padding and stroke.
+
+### Border
+
+By design, the labels do not have border. You may add border by declaring the `<SankeyLabelsBorder>` tag inside the `<SankeyLabels>` and specifying the desired settings. The  `<SankeyLabelsBorder>` provides the following parameters:
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+| Parameter | Type | Description |
+| --------- | ---- | ----------- |
+| `Color` | `string` | The color of the border. |
+| `DashType` | `DashType` enum <br/> (`DashType.Solid`) | The type of the border. |
+| `Width` | `double?` | The width of the border. |
+
+### Margin
+
+The `<SankeyLabelsMargin>` child tag provides the following properties:
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+| Parameter | Type | Description |
+| --------- | ---- | ----------- |
+| `Left` | `double?` | The left margin of the labels. |
+| `Right` | `double?` | The right margin of the labels. |
+
+### Offset
+
+The `<SankeyLabelsOffset>` child tag provides the following properties:
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+| Parameter | Type | Description |
+| --------- | ---- | ----------- |
+| `Left` | `double?` | The left offset of the labels. |
+| `Top` | `double?` | The top offset of the labels. |
+
+### Padding
+
+The `<SankeyLabelsPadding>` child tag provides the following properties:
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+| Parameter | Type | Description |
+| --------- | ---- | ----------- |
+| `Top` | `double?` | The top padding of the labels. |
+| `Bottom` | `double?` | The bottom padding of the labels. |
+| `Left` | `double?` | The left padding of the labels. |
+| `Right` | `double?` | The right padding of the labels. |
+
+### Stroke
+
+The `<SankeyLabelsStroke>` child tag provides the following properties:
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+| Parameter | Type | Description |
+| --------- | ---- | ----------- |
+| `Color` | `string` | The color of the stroke. |
+| `LineJoin` | `string` | The [line join](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/stroke-linejoin) of the stroke. |
+| `Left` | `double?` | The width of the stroke. |
+
+## Example
+
+Here is an example customization of the labels in the Sankey Diagram.
```

Comment:
Convert to a `>caption`

Reply:
This is deliberately not a caption as it is the only text in this section (similar to https://docs.telerik.com/blazor-ui/components/popover/position-collision#example)